spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -34,7 +34,7 @@
       mesh: [mesh_term],
       lcsh: [lcsh_term],
       subject_geographic: ["Boston (Mass.)", "Chicago (Ill.)"],
-      based_near: ['Boston, Massachusetts, United States', 'East Peoria, Illinois, United States'],
+      based_near: ['East Peoria, Illinois, United States', 'Boston, Massachusetts, United States'],
```

Comment:
What changed here?

Reply:
Honestly, I have no clue what happened, but for some reason the test suite expected them in reversed order on this branch. None of the objects used for testing are changed here so I'm not really sure what to think.